includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -296,7 +296,7 @@ public function generate_print_receipt( $request ) {
 			];
 
 			/* Generate receipt */
-			$response = $this->receipts_service->get_receipt_markup( $settings, $order, $charge );
+			$response = [ 'html_content' => $this->receipts_service->get_receipt_markup( $settings, $order, $charge ) ];
```

Comment:
I'm not sure if we change the response structure here, can you share details behind this change, please?

Reply:
@mgascam shared context for the change, hence ignore my previous question.